examples/Example/src/new_api/drag_n_drop/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import DragAndDrop, { DraggableItemData } from './DragAndDrop';
+import Tile, { ColorTile } from './Tile';
+
+const COLORS = [
+  '#F94144',
+  '#F3722C',
+  '#F8961E',
+  '#F9844A',
+  '#F9C74F',
+  '#90BE6D',
+  'pink',
+  '#43AA8B',
+  '#4D908E',
+  '#577590',
+  '#277DA1',
+];
+
+const COLORS_ITEMS = COLORS.map((c) => ({
+  id: c,
+  color: c,
+}));
+
+const Example = () => {
+  const handleOrderChange = (items: ColorTile[]) => {
+    // do stuff with newly ordered items
+  };
```

Comment:
Do we need it?

Reply:
this prop is required so I added handler just for showcase